controllers/comments.controller.js

```diff
@@ -47,3 +51,14 @@ exports.getComments = (req, res, next) => {
       next(err);
     });
 };
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  Promise.all([removeComment(comment_id), checkCommentExists(comment_id)])
```

Comment:
This is quite risky here as Promise.all() will fail if any of the promises fail, in case removeComment runs faster for some reason than checkCommentExists, then it will remove the item but still send an error message back to the user. 
Instead I recommend to modify the delete query slightly to make it return the deleted item within the model, then check if it's there. If not, the promise can be rejected with the 404 (still in the model).